mungegithub/mungers/submit-queue.go

```diff
@@ -800,6 +800,21 @@ const (
 	unmergeableMilestone    = "Milestone is for a future release and cannot be merged"
 )
 
+func getEarliestApprovedTime(obj *github.MungeObject) *time.Time {
+	lgtmTime := obj.LabelTime(lgtmLabel)
+	approvedTime := obj.LabelTime(approvedLabel)
+	// if both lgtmTime and approvedTime are nil, this func will return nil pointer
+	if lgtmTime == nil {
+		return approvedTime
+	} else if approvedTime == nil {
+		return lgtmTime
+	} else if lgtmTime < approvedTime {
```

Comment:
I also wonder if we are supposed to use < or if we are supposed to use *lgtmTime.Before(*approvedTime)